tests/jballerina-unit-test/src/test/resources/test-src/expressions/elvis/elvis-expr.bal

```diff
@@ -133,4 +133,30 @@ function testElvisNestedTupleTypeCaseThree() returns [string, int] {
     [string, int] dT = ["default", 0];
     var rT = xT ?: (x2T ?: dT);
     return rT;
-}
\ No newline at end of file
+}
+
+function testElvisAsArgumentPositive() returns string|error {
+    string[] filters = [];
+    json j = {};
+    string? mergable = check j.mergeable;
```

Comment:
Since `j` doesn't have a `mergeable` field, `j.mergeable` will evaluate to error and the function execution will terminate here. So we aren't actually testing the runtime aspect of the elvis expression below. 

Shall we just use something like
```suggestion
    string? mergable = ();
```